config/locales/cy/views/advice_pages/gas_costs.yml

```diff
@@ -5,7 +5,7 @@ cy:
         comparison:
           title: Cymhariaeth cost ar gyfer y 2 flynedd ddiwethaf
         cost_breakdown_by_charge:
-          title: Dadansoddiad cost yn ôl tâl am y %{ cyfnod} diwethaf
+          title: Dadansoddiad cost yn ôl tâl am y %{period}diwethaf
```

Comment:
should it have space after %{period}?

Reply:
Fixed here and 1 other place. Also updated in Transifex